lib/ex_integrate.ex

```diff
@@ -0,0 +1,16 @@
+defmodule ExIntegrate do
+  @moduledoc """
+  Documentation for `ExIntegrate`.
+  """
+
+  def run_steps(file) do
```

Comment:
I love how simple this is. 

Speculating here, I feel like this function presents a possible skeleton for an app structure. At some point, I imagine each line of this could basically be broken out into its own module, e.g.
* finding the input file
* decoding the input file with the appropriate (de)serializer
* logging the output (and our own output)
* setting up and running the external command

All of these can be extracted and tested later (and we'll have a better sense for this later).